base_classes/NXobject.nxdl.xml

```diff
@@ -30,5 +30,164 @@
+						
+						Syntax: https://igsn.org/{IGSN-ID}
+						Example: https://igsn.org/IGSN:XYZ123456
+					</doc>
+				</item>
+				<item value="ISNI">
+					<doc>
+						ISNI is an ISO standard to uniquely identify individuals and organizations involved in creative work, including pseudonyms
+						and other public personas.
+						
+						An ISNI is made up of 16 digits, the last character being a check character. The check character may be either a decimal digit
+						or the character “X”. 
+						
+						Syntax: https://isni.org/isni/{ISNI-ID}
+						Example: https://isni.org/isni/0000000121032683
+					</doc>
+				</item>
+				<item value="ISSN">
+					<doc>
+						International Standard Serial Number
+						
+						An ISSN is an 8-digit unique identifier used to distinguish a serial publication, whether in print or electronic form.
+						
+						Syntax: ISSN XXXX-XXXX
+						Example: ISSN 1234-5678
+					</doc>
+				</item>
+				<item value="ISSN-L">
+					<doc>
+						Linking ISSN
+						
+						The linking The ISSN, or ISSN-L, is a specific ISSN that groups the different media of the same serial publication.
+						
+						Syntax: ISSN-L XXXX-XXXX
+						Example: ISSN-L 1234-5678
+					</doc>
+				</item>
+				<item value="ORCID">
+					<doc>
+						Open Researcher and Contributor identifier.
+						
+						ORCID provides a free and persistent identifier that uniquely distinguishes authors and contributors in scientific research.
+						
+						Example: https://orcid.org/XXXX-XXXX-XXXX-XXXX
+						Syntax: https://orcid.org/0000-0002-1825-0097
+					</doc>
+				</item>
+				<item value="PURL">
+					<doc>
+						Persistent Uniform Resource Locator.
+						
+						A Persistent Uniform Resource Locator (PURL) is a type of URL designed to provide a stable, long-term reference to a web 
+						resource by using a resolver to redirect users to the resource's current location, even if it moves over time.
+						
+						A PURL has three parts: (1) a protocol, (2) a resolver address, and (3) a name.
+						
+						Syntax: https://purl.org/foo/bar
+						Example: https://purl.org/dc/elements/1.1/title
+					</doc>
+				</item>
+				<item value="ROR">
+					<doc>
+						Research Organization Registry
+						
+						A ROR ID is a globally unique identifier for research organizations, enabling unambiguous linking of institutions across systems.
+						
+						Syntax: https://ror.org/{ROR-ID}
+						Example: https://ror.org/052gg0110
+					</doc>
+				</item>
+				<item value="URL">
+					<doc>
+						Uniform Resource Locator
+						
+						Also known as web address, a URL is the address used to access a resource on the internet, 
+						specifying its location and the protocol to retrieve it.
+						
+						Syntax: scheme://domain:port/path?query_string#fragment_id
+						Example: https://www.example.com/about
+					</doc>
+				</item>
+				<item value="URN">
+					<doc>
+						Uniform Resource Name
```

Comment:
We probably should mention that identifiers with more specific `type` attribute values should not be stored as a URN, even when this is valid.

The two example that springs to mind are DOI and ISSN, but there may be others with valid URN namespaces.

The `doi` URN namespace [has been registered](https://www.iana.org/assignments/urn-formal/doi), so the URN `doi:10.1107/S1600576714027575` is a valid URN-based representation for the DOI `10.1107/S1600576714027575`.

Similarly, the `ISSN` URN namespace [has been registered](https://www.iana.org/assignments/urn-formal/issn), so the URN `URN:ISSN:1234-1231` is a valid URN that refers to the ISSN `1234-1231`.

However, I believe we shouldn't allow DOIs (and IGSNs) and ISSNs to be written using the `URN` type, but rather with their more specific types.